utils/RebalancingTokenWrapper.ts

```diff
@@ -0,0 +1,166 @@
+import * as _ from 'lodash';
+import { SetProtocolUtils, Address } from 'set-protocol-utils';
+
+import {
+  CoreContract,
+  CoreMockContract,
+  SetTokenContract,
+  RebalancingSetTokenContract
+} from './contracts';
+import { BigNumber } from 'bignumber.js';
+
+import { ether } from './units';
+import {
+  DEFAULT_GAS,
+  DEFAULT_PERIOD_INTERVAL,
+  DEFAULT_UNIT_SHARES,
+  DEFAULT_REBALANCING_NATURAL_UNIT,
+  DEFAULT_TIME_FAST_FORWARD
+} from './constants';
+
+import { CoreWrapper } from './coreWrapper';
+import { ERC20Wrapper } from './erc20Wrapper';
+import { Blockchain } from './blockchain';
+
+declare type CoreLikeContract = CoreMockContract | CoreContract;
+
+export class RebalancingTokenWrapper {
+  private _tokenOwnerAddress: Address;
+  private _coreWrapper: CoreWrapper;
+  private _erc20Wrapper: ERC20Wrapper;
+  private _blockchain: Blockchain;
+
+  constructor(
+    tokenOwnerAddress: Address,
+    coreWrapper: CoreWrapper,
+    erc20Wrapper: ERC20Wrapper,
+    blockchain: Blockchain,
+  ) {
+    this._tokenOwnerAddress = tokenOwnerAddress;
+
+    this._coreWrapper = coreWrapper;
+    this._erc20Wrapper = erc20Wrapper;
+    this._blockchain = blockchain;
+  }
+
+  public async createSetTokens(
```

Comment:
This doesn't seem rebalancing specific. Should this be in CoreWrapper instead?

Reply:
If we wanna move it at some point we can. Just feels like the only functionality we need multiple SetTokens for rn is rebalancing.